hnn_core/network.py

```diff
@@ -1045,6 +1047,14 @@ def isoma(self):
     def times(self):
         return self._times
 
+    def reset(self):
```

Comment:
add a tiny test?

Reply:
yes, and something to catch this in either `test_network` or `test_parallell...`.